stock_warehouse_flow/models/stock_warehouse_flow.py

```diff
@@ -176,8 +181,29 @@ def _compute_warning(self):
             except UserError as exc:
                 flow.warning = str(exc)
 
+    def _is_equal(self, flow):
```

Comment:
You can implement == operator if you also add
```
and self.warehouse_id == flow.warehouse_id
and self.from_picking_type_id == flow.from_picking_type_id
```

```suggestion
    def __eq__(self, flow):
```

Reply:
I renamed it to `_are_apply_conditions_equal`
Because a wanted to keep the odoo defaults behavior https://github.com/odoo/odoo/blob/36459d26f1adb92f92d52ce05329e8ad3e95dd91/odoo/models.py#L5636